server/db/schemas/addresses.sql

```diff
@@ -1,11 +1,18 @@
 CREATE TABLE addresses (
-    address_id serial PRIMARY KEY,
-    user_id INT NOT NULL,
+    address_id SERIAL,
     street_address VARCHAR(255) NOT NULL,
-    street_address_line_two VARCHAR(255),
+    street_address_line_two VARCHAR(255) NOT NULL,
```

Comment:
`street_address_line_two` is optional so can be null.

Reply:
If the attribute is null you can't use it in a `UNIQUE` constraint and so 2 people in an apartment building for example would have overlapping addresses.

Reply:
I figured as much but wasn't sure. Thanks for the clarification.